src/platform-apps/channels/channel-view.tsx

```diff
@@ -186,27 +186,29 @@ export class ChannelView extends React.Component<Properties, State> {
           />
         )}
         <InvertedScroll className='channel-view__inverted-scroll'>
-          <div className='channel-view__name'>
-            <h1>Welcome to #{this.props.name}</h1>
-            <span>This is the start of the channel.</span>
+          <div className='channel-view__main'>
+            <div className='channel-view__name'>
```

Comment:
The rest of this file is just indentation changes. View without whitespace changes for a nicer picture.